src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -43,20 +45,25 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
         public string ProjectKey { get; private set; }
 
         [ImportingConstructor]
-        public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolutionTracker)
+        public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolutionTracker,
+            ISonarLintOutput sonarLintOutput)
```

Comment:
I know this was already merged previously but I would have named the interface `ISonarLintLogger`.

Reply:
I don't know... Logger has different semantic - you usually put trace level in the calls, here we just write something to the window. It is similar though, we could rename in a separate commit/ticket.